articles/openshift/oshift-ref-kibana-dashboards.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Explanation of the custom dashboards we provide with the aggregated logging kibana | UKCloud Ltd
+description: Explains what our custom dashboards in Kibana show and how this data can be used.
```

Comment:
Explains the data provided in our custom Kibana dashboards and how you can use that data